utils/build-script

```diff
@@ -2018,6 +2027,11 @@ iterations with -O",
     parser.add_argument(
         "--enable-tsan-runtime",
         help="enable Thread Sanitizer on the swift runtime")
+    parser.add_argument(
+        "--enable-lsan",
+        help="enable Leaks Sanitizer for swift tools",
```

Comment:
Looks like there are still a few cases of "Leaks" instead of "Leak" ("Leak" is the existing clang convention).